packages/react/src/radio/Radio.css.ts

```diff
@@ -0,0 +1,70 @@
+import { theme } from "../styles";
+import { style } from "../vanilla-extract";
+
+export const optionWrapper = style({
+  alignItems: "center",
+  flexDirection: "row",
+  gap: "8px",
+  selectors: {
+    ["&:not(:first-child)"]: {
+      marginTop: "12px",
+    },
+  },
+});
+
+export const RadioGroupItem = style({
+  all: "unset",
+  backgroundColor: "white",
+  border: "1px",
+  borderColor: theme.colors["fg.tertiary"],
+  borderRadius: "100%",
+  borderStyle: "solid",
+  height: 16,
+  width: 16,
+
+  selectors: {
+    "&:focus-visible": {
+      outline: "2px",
+      outlineColor: theme.colors["brand.300"],
+      outlineOffset: "1px",
+      outlineStyle: "solid",
+    },
+    ["&:not(:first-child)"]: {
+      marginTop: "12px",
+    },
+    "&[data-disabled]": {
+      borderColor: theme.colors["border.secondary"],
+    },
+    "&[data-state='checked']&:not([data-disabled])": {
+      borderColor: theme.colors["brand.500"],
+    },
+  },
+});
+
+export const RadioGroupIndicator = style({
```

Comment:
name can be simplified to 
```suggestion
export const indicator = style({
```